### DISCUSSION TITLE
窗口大小自定义尺寸的高度又修改了几个像素?

### DISCUSSION THREAD

**Discussion by GTEMYee** [Meloong-Git/PCL#4004] [讨论]
如题，附图
![windowsize1442807](https://github.com/Hex-Dragon/PCL2/assets/117887221/486dfedc-1e8f-4a4f-9da8-1837d361ea95)
(↑在之前的PCL版本里会得到 1440x768 )

(↓将PCL升级到最新版后)
![display1440776](https://github.com/Hex-Dragon/PCL2/assets/117887221/cfef1275-4935-442a-b534-8e21b5d5b722)

**Comment by 3gf8jv4dv** [comment]
https://github.com/Hex-Dragon/PCL2/issues/3463#issuecomment-2130285272

**Reply by iPodtouch6557** [reply]
终于明白秦始皇车同轨的重要性了
